arm-storage/2016-01-01/swagger/storage.json

```diff
@@ -463,7 +463,7 @@
       "properties": {
         "name": {
           "type": "string",
-          "description": "Gets or sets the sku name. Required for account creation, optional for update. Note that in older versions, sku name was called accountType.",
+          "description": "Gets or sets the sku name. Required for account creation, optional for update. Note that in older versions, sku name was called accountType. For Blob Storage accounts, only Standard_LRS, Standard_GRS and Standard_RAGRS are supported",
```

Comment:
> Required for account creation, optional for update.
This statement is weird. 

Does that mean that during update, someone can pass an empty sku object `{}`?
/cc @johanste @anudeepsharma